src/library/dlgcoverartfullsize.cpp

```diff
@@ -46,6 +46,12 @@ DlgCoverArtFullSize::DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlay
     setupUi(this);
 }
 
+void DlgCoverArtFullSize::closeEvent(QCloseEvent* event) {
+    // prevent the window from being destroyed
```

Comment:
Please also note here why this workaround is required, instead of only in the description of this PR.